test/sets.jl

```diff
@@ -322,6 +322,16 @@ function test_sets_Interval()
     @test MOI.Interval(MOI.Interval(1, 2)) == MOI.Interval(1, 2)
 end
 
+function test_sets_reified()
+    set = MOI.Reified(MOI.AllDifferent(2))
+    @test MOI.dimension(set) == 3
+    @test set == copy(set)
+    set = MOI.Reified(MOI.GreaterThan(1))
+    @test MOI.dimension(set) == 2
+    @test set == copy(set)
+    return
```

Comment:
The `return` is not present for the three functions above, maybe that should be fixed?

Reply:
Yeah, some functions that end in a macro call are missing a `return` statement, because the formatter cannot automatically infer that `return @set == copy(set)` or a blank `return` is appropriate. 

I fix them when convenient, but it's not a big deal.